hack/lib/golang.sh

```diff
@@ -232,9 +232,20 @@ kubeedge::golang::build_binaries() {
   for bin in ${binaries[@]}; do
     echo "building $bin"
     local name="${bin##*/}"
-    set -x
-    go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -gcflags="${gogcflags:-}" -ldflags "${goldflags:-}" $bin
-    set +x
+    # if env BUILD_WITH_CONTAINER exist, we build with docker images
+    set +u
```

Comment:
Remove the set +/-u?

Reply:
cannot, if want to check the variable exist or not using `if [ ! -z ${BUILD_WITH_CONTAINER} ]`, we must set +u, or will report the error `BUILD_WITH_CONTAINER: unbound variable`
and there're lots of `set -o nounset` in the scripts, this will prevent we check the env exist or not.

Reply:
refactored, and more common, PTAL